src/pages/webhooks/admin-configuration.md

```diff
@@ -21,11 +21,63 @@ The rows of this grid show configuration settings for all registered hooks, both
+Click **Add New Webhook** from the grid page to display the form for creating a new hook.
 
 <InlineAlert variant="warning" slots="text" />
 
-On Cloud instances, due to the read-only file system, the `webhooks:generate:module` command cannot be run. If a plugin-type webhook is added through the Admin, the method name and type should be declared in a `webhooks.xml` file so that the required plugin is generated during the build phase of the deployment process.
+If the plugin for the webhook method and type entered into the form have not been generated for the Commerce instance, a warning to run the `webhooks:generate:module` command will appear upon clicking **Save**. You cannot run the `webhooks:generate:module` command on Cloud instances, due to its read-only file system. If you add a plugin-type webhook through the Admin, declare the method name and type in a `webhooks.xml` file.
+
+![New webhook](../_images/webhooks/new-hook-settings.png)
+
+The **Hook Settings** configuration panel contains the following fields:
+
+Field | Description
+--- | ---
+**Webhook Method** | The internal name of a webhook. The value must be in the form `<type>.<webhook_name>`, where `type` is either `observer` or `plugin`, and `webhook_name` matches a valid Commerce event name. Use the [`bin/magento webhooks:list:all`](commands.md#return-a-list-of-supported-webhook-event-names) command to display a list of possible webhooks.
+**Webhook Type** | Select whether to run the webhook `before` or `after` the original action.
+**Batch Name** | A unique name for the batch. Use a descriptive name that encompasses all the hooks in the batch. The name must contain English alphanumeric characters and underscores (_) only.
+**Hook Name** |  A name that must be unique within a batch. The name must contain English alphanumeric characters and underscores (_) only.
+**URL** | The HTTP endpoint to send the request for processing.
+**Timeout** | A hard timeout limit (milliseconds) for the request. Requests exceeding this timeout are aborted and logged. The default value of 0 indicates there is no timeout limit.
+**Soft timeout** | A soft timeout limit (milliseconds) for the request. Requests exceeding this timeout are logged for debugging purposes.
+**Cache TTL** | The cache time-to-live (in seconds) for requests with the same URL, body, and headers. If this attribute is not specified, or if the value set to `0`, the response is not cached.
+**Fallback error message** | The error message to display when the hook fails.
+**Required** | Specifies whether hook execution is required or optional. When set to `Optional`, if the hook fails to execute, the failure is logged and subsequent hooks continue to be processed. When set to `Required`, a failure terminates the process.
+**Active** | Indicates whether to skip a removed hook during the batch execution.
+**Method** | The HTTP method (POST, PUT, GET, or DELETE) used to invoke the hook.
+
+### Configure hook fields
```

Comment:
One idea from Sasha was to show screenshots of each of the fields/headers/rules section. For fields though, there is already a screenshot later in the `Edit an existing hook` section, but it's for the case when there are xml-defined hook fields that can't be edited

Reply:
I decided against adding screenshots for the bottom half of the configuration screen. The UI is repetitive and I don't think adding the screenshots really adds anything.